src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -893,6 +894,8 @@ inlineCommands = M.union inlineLanguageCommands $ M.fromList
   , ("cref", rawInlineOr "cref" $ doref "ref")       -- from cleveref.sty
   , ("vref", rawInlineOr "vref" $ doref "ref+page")  -- from varioref.sty
   , ("eqref", rawInlineOr "eqref" $ doref "eqref")   -- from amsmath.sty
+  , ("mbox", rawInlineOr "mbox" $ spanWith ("",["mbox"],[]) . dropLineBreaks <$> tok)
+  , ("hbox", rawInlineOr "hbox" $ spanWith ("",["hbox"],[]) . dropLineBreaks <$> tok)
```

Comment:
What's the rationale for `dropLineBreaks` here?

Reply:
`hbox` stands for «Horizontal box» in TeX. They do not only drop the line breaks here but they also prohibit any line wraps inside the box. `mbox` is some kind of LaTeX analog for `hbox`.

I made a few examples in this screenshot.
The first two lines stand for explicit line breaks inside `mbox` and `hbox`, respectively.
The third contains an explicit line break inside an `Emph`, this is for another comment.
The last is a very long paragraph completely inside `mbox`, so one is not able to see its end because it doesn't fit into one line.

![Screenshot from 2019-06-16 09-16-46](https://user-images.githubusercontent.com/11293747/59560854-97afb380-9018-11e9-9fea-8879f0ea4171.png)